Examples/Run/Digitization/Common/DigitizationExample.cpp

```diff
@@ -21,7 +21,6 @@
 #include "ActsExamples/Io/Csv/CsvOptionsWriter.hpp"
 #include "ActsExamples/Io/Csv/CsvParticleReader.hpp"
 #include "ActsExamples/Io/Csv/CsvSimHitReader.hpp"
-#include "ActsExamples/Io/Json/JsonDigitizationConfig.hpp"
```

Comment:
Again, I am a bit uneasy about moving the `JsonDigitizaitonConfig` into the `Algorithm/Digitization`.

Reply:
See above